yt_idefix/data_structures.py

```diff
@@ -881,10 +882,9 @@ def _parse_parameter_file(self):
             # so we normalize standard output field names to upper case
             # to avoid duplicating data in PlutoFields.known_other_fields
             for varname in self._detected_field_list:
-                if (
-                    varname.upper()
-                    in unzip(self._field_info_class.known_other_fields)[0]
-                ):
+                if varname.upper() in [
+                    f[0] for f in self._field_info_class.known_other_fields
+                ]:
```

Comment:
we should still store the result of the list comp rather than recomputing it for every iteration of the parent for loop

Reply:
Got it！